StormyHares/BangazonApi
# Models mw

## Ticket Link:
  - #17

## Description:
  - Modeled DB and created migration

## Steps to Test:

  - Dote Net Restore to update packages
  - Check to make sure you have an environment variable IE : 
export BANGAZON="/Users/camiller/workspace/Bangazon/BangazonApi/BangazonApi.db"
  - dotnet ef database update
